app/import_functions.go

```diff
@@ -784,7 +784,7 @@ func (a *App) ImportReply(data *ReplyImportData, post *model.Post, teamId string
 		if err != nil {
 			return err
 		}
-		reply.FileIds = fileIds
+		reply.FileIds = append(reply.FileIds, fileIds...)
```

Comment:
I have very little insight into the import feature, but is there an expectation of idempotency? That is, if I run the bulk importer more than once, am I going to now end up with duplicate attachments? Should we deduplicate somehow?